src/controller/java/templates/templates.json

```diff
@@ -24,10 +24,6 @@
             "name": "cluster_header",
             "path": "../../../app/zap-templates/partials/cluster_header.zapt"
         },
-        {
```

Comment:
This is somewhat unrelated & really more just cleanup: this file no longer exists